snap/local/README.md

```diff
@@ -7,9 +7,9 @@
 
 # Building and testing the snap
 
-Make sure that Convos is checked out in your `$HOME` directory:
-
-    $ ./snap/build.sh
+    - Install snapcraft `sudo snap install snapcraft --classic`
+    - Clone convos repo
+    - Run `snapcraft` to build the snap (this will install multipass for vm based building)
```

Comment:
Nitpicking, but this looks a bit inconsistent. Can you remove the spaces in front of "-" in the list?

I guess the "Make sure that Convos is checked out in your `$HOME` directory" statement is completely outdated?

Reply:
Yea you don't need this to be specifically checked out in `$HOME`